sklearn/ensemble/_bagging.py

```diff
@@ -138,9 +138,10 @@ def _parallel_predict_proba(estimators, estimators_features, X, n_classes):
         else:
             # Resort to voting
             predictions = estimator.predict(X[:, features])
-
-            for i in range(n_samples):
-                proba[i, predictions[i]] += 1
+            # Lookup for converting class labels to column numbers
+            to_index = dict(zip(estimator.classes_, itertools.count()))
+            # Increment label counts for this estimator
+            proba[range(n_samples), [to_index[i] for i in predictions]] += 1
```

Comment:
I think that we can simplify using broadcasting without using a dictionary as a lookup table.

```suggestion
            # Find classes indices of the predictions
            class_indices = np.where(predictions[:, None] == estimator.classes_[None, :])[1]
            proba[:, class_indices] += 1
```

Reply:
That works too. Might be more memory intensive?

Reply:
> I believe proba[:, class_indices] += 1 increments the whole row. I don't think that is the intended behavior here

Yep this is probably the most important part ;)

Reply:
I probably mess-up. I wanted to find a way that does not require look-up table.